docs/portfolios/Basics.md

```diff
@@ -4,9 +4,36 @@ tags: [Portfolios]
 
 # Portfolio basics
 
-Portfolios are a container for an investor's cash and holdings. All investment activity on the WealthKernel platform (balances, orders, transactions) is directly tied to a portfolio.
+Portfolios are a container for an investor's cash and holdings. All investment activity on the WealthKernel platform (balances, orders, transactions) is directly tied to a portfolio. When a portfolio is created under an account, it will inherit the same currency as the account. Multiple portfolios can be created under a single account to allow for different investment strategies. However, we do not recommend mixing and matching mandate types.
 
-You can have multiple portfolios under a single account to allow for different investment strategies. However, we do not recommend mixing and matching mandate types.
+The below diagram represents an example account and portfolio structure. 
+
+
+```mermaid
+flowchart TD
+    A(Party - Person) --> B(GIA GBP) --> G(Portfolio GBP)
+    B(GIA GBP) --> H(Portfolio GBP)
+    A(Party - Person) --> C(GIA EUR) --> I(Portfolio EUR)
+    A(Party - Person) --> D(GIA USD) --> J(Portfolio USD)
+    A(Party - Person) --> E(ISA) --> K(Portfolio)
+    A(Party - Person) --> F(SIPP) --> L(Portfolio)
+    F(SIPP) --> M(Portfolio)
+```
+## Portfolio characteristics
+
+The following table provides a summary of how currencies work across different functions. 
+
+| Function | GBP Portfolio | EUR Portfolio | USD Portfolio | Notes |
+|---|---|---|---|-------------|
+| Money in | GBP, EUR, USD | GBP, EUR, USD | GBP, EUR, USD |All currencies can be paid into any portfolio under a GIA account. Portfolios under ISA and SIPP accounts only support GBP being paid in|
+| Money out | GBP, EUR, USD | GBP, EUR, USD | GBP, EUR, USD |All currencies can be paid out of any portfolio under a GIA account. Portfolios under ISA and SIPP accounts only support GBP being paid out|
+| Valuations | GBP | EUR | USD |The valuation of a portfolio will be in the currency of the account that the portfolio is under|
```

Comment:
Strictly speaking we drive the valuations and balances off the portfolio.Currency, same as balances.  Suggest the Notes follow the same style as the Note for balances, e..g. The valuation of a portfolio will be in the currency of the portfolio.